### PR TITLE
Simplify Oracle CI testing

### PR DESCRIPTION
**Pull request by mvorisek**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| Fixed issues | n/a

#### Summary

As discussed in https://github.com/doctrine/dbal/pull/6217#discussion_r1383558731 thanks to https://github.com/gvenzl/oci-oracle-xe/issues/238.

### CODE REVIEW

**Comment by derrabus** [comment]
Thanks. Converted to draft as long as the CI is failing.

**Comment by mvorisek** [comment]
Thanks. Any idea why? In https://github.com/atk4/data/pull/1150/commits/6c59956de2a72cce3ea2d65d3d456e3fd59f0e8f the `FREE` service name for v21 is working.

**Comment by derrabus** [comment]
I don't really have time to investigate this right now, sorry. 😓

File: src/Driver/AbstractOracleDriver/EasyConnectString.php [hunk 1]
```diff
@@ -55,15 +55,7 @@ public static function fromConnectionParameters(array $params): self
         $connectData = [];
 
         if (isset($params['servicename']) || isset($params['dbname'])) {
```
Comment (mvorisek) [inline src/Driver/AbstractOracleDriver/EasyConnectString.php:57]: Based on:
- https://github.com/gvenzl/oci-oracle-xe/issues/238#issuecomment-1920591949
- https://github.com/gvenzl/oci-oracle-free/issues/34

the service name should be the default, SID is internal DB identifier, but service name is more or less like "DB name", does support aliasing or clustering.

**Comment by mvorisek** [comment]
Based on https://github.com/gvenzl/oci-oracle-xe/issues/238#issuecomment-1920591949 it is not possible iwthout switching from SID to "service name".

**Pull request closed without merging**